
## NO-RELATIVE
### Rule name: `no-relative`

#### Rule Details
æ€»æ˜¯åœ¨webpackæˆ–è€…å…¶ä»–æ„å»ºå·¥å…·ä¸­é…ç½®alias, è€Œä¸æ˜¯ä½¿ç”¨å¤šå±‚çº§çš„ç›¸å¯¹è·¯å¾„

ğŸ‘ŒGOOD
```javascript
import bar from 'foo/baz.ext';,
import bar from './foo.js';,
import bar from '@/foo.js';,
import { lastName as surname } from './profile.js';,
import { a } from './xxx.js',
import 'lodash';,
import * as circle from './circle';,
```

ğŸ˜¢BAD
```javascript
import a from '../a.js';
import bb, { cc } from './../../../../xx.js';
import e from './.e.js';
import lodash from 'lodash';
import foo, { lastName as surname } from '../profile.js';
import { aa } from '../../../xxx.js';
import '../../lodash';
import * as circle from '../circle';
```
### é”™è¯¯æç¤º

#### vscode
  - <img src="//meipu1.video.meipai.com/9a086867-ddfa-48b7-91f7-fc88d7f55f86.png" width="800px" />

#### cli
- <img  src="//meipu1.video.meipai.com/9ddfae12-9404-4439-9c5f-80cd367555fe.png" width="800px" />
